mxcubeweb/core/components/queue.py

```diff
@@ -2481,3 +2487,10 @@ def set_setting(self, name_value: SimpleNameValue) -> tuple:
             result = ()
 
         return result
+
+    def set_num_snapshots(self, num_snapshots: int):
+        """Sets the number of snapshots to take during data collection
+        Args:
```

Comment:
Change `Args` to `Parameters`.

This is the mark-up style our sphix doc generator expects here.

Reply:
The list of sections is here: <https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections>.

`Args`, and `Arguments` are aliases for `Parameters`.

I do not know that we have a policy of picking one name over the others.

Reply:
Changed in the latest push, but I am a bit confused.
https://mxcubeweb.readthedocs.io/en/latest/dev/contributing.html#style-guidelines
here it says that `Args` should be used and searching for string `'Parameters:'` in the codebase did not yield any results in python files.

Reply:
Indeed, we agreed to use Google style, and now that I check it again the Google style seems to be recommending `Args`: <https://google.github.io/styleguide/pyguide.html#doc-function-args>. So let's go with `Args`. Hopefully at some point we will introduce a linter that checks and enforces the docstrings format for us (because we keep forgetting).

Reply:
I did not know that `Args` and `Arguments` also work, TIL!

Then *I* was doing this wrong. `Args` is fine then, doh.